Packages/io.chainsafe.web3-unity/Runtime/Scripts/ClipboardManager.cs

```diff
@@ -37,22 +35,7 @@ private void InitializeClipboardHandler()
             _clipboardHandler = null;
 #endif
         }
-
-        private void Update()
-        {
-#if ENABLE_INPUT_SYSTEM
-            if ((Keyboard.current.leftCtrlKey.isPressed || Keyboard.current.leftCommandKey.isPressed) && Keyboard.current.vKey.wasPressedThisFrame)
-            {
-                _clipboardHandler?.Paste();
-            }
-#else
-            if ((Input.GetKey(KeyCode.LeftControl) || Input.GetKey(KeyCode.LeftCommand)) && Input.GetKeyDown(KeyCode.V))
-            {
-                _clipboardHandler?.Paste();
-            }
-#endif
-        }
-
+        
```

Comment:
This is redundant since there is not a single place in our SDK where users can paste things.

Reply:
I think @sneakzttv put it in to paste on WebGL using the Web3Auth Wallet GUI, for pasting address for transactions

Reply:
No that was mostly done for pasting things in the text input fields, but we don't have them anywhere in our sdk anymore

Reply:
we have them in the Web3Auth wallet GUI, the token and address input fields

Reply:
You are right. I will revert it!